composer.json

```diff
@@ -71,10 +71,10 @@
         "symfony/expression-language": "^3.4 || ^4.0",
         "symfony/finder": "^3.4 || ^4.0",
         "symfony/form": "^3.4 || ^4.0",
-        "symfony/framework-bundle": "^4.2",
+        "symfony/framework-bundle": "^4.3",
         "symfony/mercure-bundle": "*",
-        "symfony/messenger": "^4.2",
-        "symfony/phpunit-bridge": "^4.3",
+        "symfony/messenger": "^4.3",
```

Comment:
Is it a good idea to bump the dependencies in a patch version? It means that our (potential) future security fixes will not reach the users who are stuck on `<=2.4.4`

Reply:
What this essentially means:

You must upgrade Symfony Messenger to continue to receive security fixes for 2.4

Reply:
It's not for a patch version but a minor one. I think it's OK because this component is experimental in Symfony anyway.

Reply:
Uhh... Didn't notice this was targeting `master` 🙈